Sources/StreamChat/Controllers/ChannelListController/ChannelListController.swift

```diff
@@ -89,6 +89,7 @@ public class ChatChannelListController: DataController, DelegateCallable, DataSt
                 }
                 log.debug("didChangeChannels: \(changes.map(\.debugDescription))")
                 $0.controller(self, didChangeChannels: changes)
+                $0.controller(self, didChangeState: state)
```

Comment:
The only potential risk here is that this changes how the SDK works at the moment 🤔 The `didChangeState` will get called everytime whenever there are data updates, which did not happen before 🤔 So it might be worth having some logic to call this only on the first page when remoteDataFetched 🤔

Reply:
Theoretically, it shouldn't change much, I think. We will just have more state updates with the same state. However, to be safe, maybe we can add more conditions here, like first page + background mapping enabled.

Reply:
I have been looking at this for some time and the only thought is that since all of this is related to background mapping then should we only do this when background mapping is enabled. If background mapping gets enabled for everyone at some point then we could just clean up that check. Less risk that additional `didChangeState` would degrade customer's implementation somehow. 🤔

Reply:
We agreed on doing this change only with BG Mapping enabled 👍